ebl/src/main/java/org/dcsa/conformance/standards/ebl/checks/EBLChecks.java

```diff
@@ -666,6 +668,19 @@ private static Consumer<MultiAttributeValidator> allDg(Consumer<MultiAttributeVa
               JsonAttribute.mustBeAbsent(SI_REQUEST_SEND_TO_PLATFORM)),
           JsonAttribute.mustBeAbsent(SI_REQUEST_SEND_TO_PLATFORM));
 
+  static final JsonRebaseableContentCheck VALID_PARTY_FUNCTION =
+      JsonAttribute.allIndividualMatchesMustBeValid(
+          "The partyFunction is valid",
```

Comment:
Since there are two partyFunctions, please change the message like "The partyFunction in other DocumentParties is valid".

Reply:
One is at the root level and one inside hbl (but both inside otherDocumentParty) so updated the message accordingly.